HuffmanCoding/HuffEncode.h

```diff
@@ -31,31 +32,31 @@ struct Node {
 		
 	}
 };
+typedef Node* node_ptr;
 
 
 
 
 class HuffEncode {
 public:
-
+	std::map<char, std::string> stored_map;
 	HuffEncode();
-	void startEncoding(std::string);
-	void writeBit(int bit);
+	~HuffEncode();
+	void startEncoding(const std::string&);
 	void writeToFile(unsigned char&);
-	std::map<std::string, std::string> stored_map;
 	void orderedMap(std::string);
-	void writeDict(std::map<std::string, std::string> &stored_map);
-	
-	
+	void writeDict(std::map<char, std::string> &stored_map);
+
 	
 
 private:
 	
 	int  bitcount;
 	unsigned char bit_buffer;
-	void build_tree(std::set<Node> &s);
-	void traverse(const Node *node,std::string str, std::map<std::string, std::string> &stored_map);
 	std::set<Node> nodes_;
+	void build_tree(std::set<Node> &s);
```

Comment:
Follow a single coding convention. Other functions are camel case, but this function is stl case.